x-pack/plugins/cases/public/components/connector_selector/form.tsx

```diff
@@ -12,9 +12,18 @@ import { css } from '@emotion/react';
 
 import type { FieldHook } from '@kbn/es-ui-shared-plugin/static/forms/hook_form_lib';
 import { getFieldValidityAndErrorMessage } from '@kbn/es-ui-shared-plugin/static/forms/hook_form_lib';
+import { i18n } from '@kbn/i18n';
 import type { ActionConnector } from '../../../common/types/domain';
 import { ConnectorsDropdown } from '../configure_cases/connectors_dropdown';
 
+const ADD_CONNECTOR_HELPER_TEXT = i18n.translate(
+  'xpack.cases.connectorSelector.addConnectorHelperText',
+  {
+    defaultMessage:
+      'Go to Stack Management > Cases > Settings to add an external incident management system',
```

Comment:
Different solutions have different paths for Cases. Maybe we can say `Go to Cases > Settings to add an external incident management system`. Wdyt?

Reply:
done, thanks 👍